sound/soc/sof/ipc3.c

```diff
@@ -316,6 +318,105 @@ static int sof_ipc3_tx_msg(struct snd_sof_dev *sdev, void *msg_data, size_t msg_
 	return ret;
 }
 
+static int sof_ipc3_set_get_data(struct snd_sof_dev *sdev, void *data, size_t data_bytes,
```

Comment:
I guess this is also based on existing implementation, just restructured? Might be good to mention in the git commit message.